### PR TITLE
add babel-polyfill

### LINKED ISSUE #395: Object.entries is not a function
https://sentry.prod.mozaws.net/operations/send-prod/issues/632248/

### PR DESCRIPTION
**Pull request by dannycoates**
fixes #395

and before @pdehaan mentions webpack, i know, but not right now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +80,7 @@
     "build:upload": "browserify frontend/src/upload.js -g uglifyify -o public/upload.js",
     "build:download": "browserify frontend/src/download.js -g uglifyify -o public/download.js",
     "build:version": "node scripts/version",
-    "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
+    "build:3rd": "cp node_modules/l20n/dist/web/l20n.min.js public && cp node_modules/babel-polyfill/dist/polyfill.min.js public",
```
Comment (pdehaan) [inline package.json]: I'll give you a 💵 if we rename this to something more grep-able like "build:vendor".

You could even combine the two:

```sh
cp node_modules/l20n/dist/web/l20n.min.js node_modules/babel-polyfill/dist/polyfill.min.js public
```

**Comment by pdehaan** [comment]
Do we know where the `Object.entries()` is coming from? I didn't see it in our source (via a quick `git grep`).

I mean, **babel-polyfill** seems like a big [96KB] hammer for an API call:

```sh
$ ls -lash node_modules/babel-polyfill/dist/polyfill.min.js | awk '{ print $6, $10 }'

96K node_modules/babel-polyfill/dist/polyfill.min.js
```

**Comment by dannycoates** [comment]
> I mean, babel-polyfill seems like a big [96KB] hammer for an API call

it is, but this isn't the only error in sentry the polyfill will fix

**Comment by pdehaan** [comment]
> it is, but this isn't the only error in sentry the polyfill will fix

👍 Great point! Ship it. I think we were considering doing webpack+babel+transpiling (#359) anyways, so this is a great start.

**Pull request merged**